src/js/pages/CheckoutCompletePage.js

```diff
@@ -17,19 +19,21 @@ export default class CheckoutCompletePage extends Component {
         <AppHeader navigation={this.props.navigation}>
           <Image source={require('../../img/peek.png')} style={styles.peek_img} />
           <View style={styles.secondary_header}>
-            <Text style={styles.header_title}>Checkout Complete!</Text>
+            <Text style={styles.header_title}>{ i18n.t('checkoutCompletePage.header')}</Text>
           </View>
-          <ScrollView style={styles.container}>
+          <ScrollView style={styles.container} keyboardShouldPersistTaps="handled" { ...testProperties(i18n.t('checkoutCompletePage.screen')) }>
 
             <View style={styles.complete_container}>
-              <Text style={styles.complete_title}>THANK YOU FOR YOUR ORDER</Text>
-              <Text style={styles.complete_text}>Your order has been dispatched, and will arrive just as fast as the pony can get there!</Text>
+              <Text style={[styles.complete_title, styles.upperCaseText]}>{ i18n.t('checkoutCompletePage.completeContainer.header')}</Text>
```

Comment:
I think the styles.upperCaseText is a leftover from a previous version of this PR and can be removed

Reply:
Fixed it, thanks!